go.mod

```diff
@@ -1,12 +1,13 @@
 module github.com/palantir/go-baseapp
 
 go 1.21
+toolchain go1.23.0
```

Comment:
Hmm why did you do this @dependabot

Reply:
https://github.com/gorilla/sessions/commit/ff5660f3c355c08371621b642d76c7a4c88836c6